Description: when Qt is built with -no-webkit and Qt WebKit is used
             Qt applications cannot found webkit as they rely on qmake. 
Author: Simon Hausmann <simon.hausmann@nokia.com>
Author: Fathi Boudra <fathi.boudra@nokia.com>

--- a/WebKit/qt/qt_webkit_version.pri
+++ b/WebKit/qt/qt_webkit_version.pri
@@ -1,3 +1,7 @@
+#configuration
+QT_CONFIG *= webkit
+
+#versioning
 QT_WEBKIT_VERSION = 4.7.0
 QT_WEBKIT_MAJOR_VERSION = 4
 QT_WEBKIT_MINOR_VERSION = 7
